Sources/HTTPParser/http_parser.swift

```diff
@@ -254,17 +390,18 @@ public enum http_errno: String, Error {
   }
 }
 
-public class http_parser {
+private func LIKELY(_ X: Bool) -> Bool { return X }
```

Comment:
I tried this on a number of functions like `LOWER(), IS_ALPHA(), IS_NUM(), IS_ALPHANUM()` and I did not see any performance improvement in Release mode.